frontend/mulighetsrommet-veileder-flate/src/components/joyride/OversiktenJoyride.tsx

```diff
@@ -0,0 +1,158 @@
+      content: 'Nå skal vi vise dere masse nye kule ting!',
+      placement: 'center',
+      target: 'body',
+    },
+    {
+      target: '#gjennomforinger-liste',
+      content: 'Her finner du alle tiltaksgjennomføringer for din NAV-enhet.',
+      placement: 'auto',
+      disableBeacon: true,
+    },
+    {
+      target: '#filtertags',
+      content: 'Listen er allerede filtrert på NAV-enhet og brukerens innsatsgruppe...',
+      placement: 'auto',
+      disableBeacon: true,
+    },
+    {
+      target: '#tiltakstype_oversikt_filtermeny',
+      content: '...men du kan også filtrere på andre ting som tiltakstyper og lokasjon!',
+      placement: 'auto',
+      disableBeacon: true,
+    },
+    {
+      target: '#sortering-select',
+      content: 'I tillegg kan du sortere listen her.',
+      placement: 'auto',
+      disableBeacon: true,
+    },
+    {
+      target: '#historikkBtn',
+      content: 'Her kan du se hvilke tiltak brukeren har vært på tidligere.',
+      placement: 'left',
+      disableBeacon: true,
+    },
+    {
+      content: 'Her kan du se historikken til brukeren i kontekst.',
+      placement: 'top',
+      styles: {
+        options: {
+          zIndex: 10000,
+        },
+      },
+      target: '#historikk_modal',
+      disableBeacon: true,
+    },
+    {
+      content: 'Her kan du lese mer om tiltaksgjennomføringene. Klikk på raden for å se!',
+      placement: 'top',
+
+      target: '#list_element_0',
+      disableBeacon: true,
+    },
+  ];
+
+  const lastStep: Step[] = [
+    {
+      target: '#joyride_knapp',
+      content: 'Hvis du vil se dette igjen kan du klikke her!',
+      placement: 'auto',
+      disableBeacon: true,
+    },
+  ];
+
+  const handleJoyrideCallback = (data: any) => {
+    const { action, index, status, type } = data;
+    const nextStepIndex = index + (action === ACTIONS.PREV ? -1 : 1);
+
+    if ([EVENTS.STEP_AFTER].includes(type)) {
+      //for steps i historikk-modalen
+      if (EVENTS.TARGET_NOT_FOUND && index === 5) {
+        setState(prevState => ({ ...prevState, run: false, loading: true }));
+        setTimeout(() => {
+          setState(prevState => ({ ...prevState, run: true, loading: false }));
+        }, 200);
+      }
+      setState(prevState => ({ ...prevState, stepIndex: nextStepIndex }));
+    } else if ([STATUS.FINISHED, STATUS.SKIPPED].includes(status)) {
+      // if ([STATUS.SKIPPED].includes(status)) {
+      if (window.localStorage.getItem('joyrideOversiktenLastStep') === null) {
+        window.localStorage.setItem('joyrideOversiktenLastStep', 'true');
+      }
+      // }
+      setState(prevState => ({ ...prevState, run: false, stepIndex: 0 }));
+      window.localStorage.setItem('joyrideOversikten', 'false');
+    }
+    // åpne historikk-modalen
+    if ([ACTIONS.NEXT] && index === 6) {
+      toggleHistorikkModal(true);
+    }
+
+    if ([ACTIONS.NEXT] && index === 7) {
+      toggleHistorikkModal(false);
+    }
```

Comment:
Som Sondre har vært inne på allerede så vil vel disse index-sjekkene brekke med en gang du legger til et step foran det som er forventet for øyeblikket.